README.md

```diff
@@ -57,8 +57,12 @@ Node (22.9.0 or higher) and npm
    - **DO NOT COMMIT** this file. (thanks!)
 
 ## Using cypress testing
+Once done creating an end-to-end test in client/cypress/e2e,
 
-1. Once done creating an end-to-end test in client/cypress/e2e, view the test work using `npx open cypress` in the client folder. Make sure the application is already running.
+1. Stop the server. Inside server/app/, export mongo db creds:
+export MONGO_URI="mongodb+srv://<username>:<password>@animus.piv6r.mongodb.net/?retryWrites=true&w=majority&appName=Animus"
```

Comment:
I dont think we need this. Server will keep the auth on

Reply:
nahi bina iske creds ke liye run mein error deta hai